src/box/lua/config/applier/runtime_priv.lua

```diff
@@ -0,0 +1,64 @@
+local function apply_rt_access(config_module)
+    local configdata = config_module._configdata
+    local credentials = configdata:get('credentials') or {}
+    credentials.roles = credentials.roles or {}
+    credentials.users = credentials.users or {}
+
+    local role_lua_call_priv = {}
+    local user_lua_call_priv = {}
+
+    local function process_lua_call(privs, user_or_role_name, user_or_role_data)
+        privs[user_or_role_name] = {}
+        local user_roles = user_or_role_data.roles or {}
+
+        if user_or_role_data.privileges then
+            for _, privilege in pairs(user_or_role_data.privileges) do
```

Comment:
Nit: AFAIR, it is an array-like table, we can use `ipairs` here.

Reply:
Done.